packages/common/src/configuration/service-configuration.ts

```diff
@@ -94,7 +101,24 @@ export class ServiceConfiguration {
                     doc: 'Timeout value after which the task has to be terminated',
                 },
             },
-
+            jobManagerConfig: {
+                targetQueuedTasksOverloadRatio: {
+                    format: Number,
+                    default: 3,
+                    // tslint:disable-next-line: max-line-length
+                    doc: `The target overload ratio of running to queued tasks. Higher ratio value will result higher queued tasks count.`,
+                },
+                tasksIncrementIntervalInSeconds: {
```

Comment:
>tasksIncrementIntervalInSeconds [](start = 16, length = 31)

addTaskIntervalInSec? or addTaskIntervalInSeconds?

Reply:
Renamed to addTasksIntervalInSeconds